solution/0500-0599/0503.Next Greater Element II/Solution.ts

```diff
@@ -0,0 +1,12 @@
+function nextGreaterElements(nums: number[]): number[] {
+    let stack: number[] = [], len = nums.length;
+    let res: number[] = new Array(len).fill(-1);
```

Comment:
对于对象，请尽量使用 `const` 声明：

```suggestion
    const stack: number[] = [], len = nums.length;
    const res: number[] = new Array(len).fill(-1);
```